modules/ROOT/pages/api-contracts-landing-page.adoc

```diff
@@ -1,15 +1,59 @@
-= Client Applications and Contracts
+= Client Applications, Contracts, and Credentials
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:keywords: api, contract, register, client, application, secret
+:keywords: api, contract, register, client, application, secret, credential
+
+Credentials consist of two keys: the client ID (also referred to as _username_) and a client secret (also referred to as _password_). Anypoint API Manager (API Manager) uses credentials in several different ways. This document presents an overview of the following:
+
+* Client Application Credentials
+* Organization Credentials
+* Basic Authentication Policy Credentials
+* SOAP API Proxy Credentials
+
+== Client Application Credentials
 
 As the Organization Administrator, after you grant access to applications in Anypoint Exchange (Exchange), a contract is automatically created between the API instance and the application. The contract gives access to the application based on SLA tiers that you might specify. Only one contract can exist per API instance and application at any time.
 
 As a user, when you request access to an API from Exchange, your request is automatically approved if the API does not include SLAs. If the API includes an SLA tier, you either configure your request so that it is automatically approved despite the SLA tier, or you can request approval manually.
 
+Client application credentials are automatically generated when your application access is approved. API requests must include these credentials if the API is protected by a client ID enforcement policy. Enforcement policies can be configured to accept credentials either as headers or as query parameters, though using headers is more secure.
```

Comment:
Thanks for clarifying. I edited the sentence on creating new applications/credentials to reflect your correction. 

I didn't include mention of the scenario where a customer requests access to an API using an existing application, because I wanted to focus on when the credentials are actually created. However, let me know if you think I should include this info.

Reply:
I think now it is correct and clear enough. Thanks!